simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,33 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(binaryList: List[int]) -> List[int]:
+    lenLst = len(binaryList)
+    
+    if lenLst == 0:
+        return -1
+    
+    for digit in binaryList:
+        if digit != 0 and digit != 1:
+            return -1
+
+    digit = 0
```

Comment:
i = 0. You should know when to use a name or when to just use "i" as iterable or index.